resources/views/partials/image-text.blade.php

```diff
@@ -0,0 +1,84 @@
+<div class="mx-auto mb-12 flex max-w-7xl flex-col gap-20 px-6 md:mb-32 md:gap-28">
+    <div class="grid items-center gap-x-60 gap-y-6 md:grid-cols-2">
+        <div class="flex flex-col items-start max-md:order-last">
+            <span class="text-sm text-secondary-100">
+                Why did we start with Rapidez
+            </span>
+            <h2 class="mt-2 text-4xl font-semibold text-heading">
+                We became tired of using the impossible front-end of Magento
+            </h2>
+            <span class="mt-8 text-lg text-inactive">
+                Lorem ipsum dolor sit amet consectetur adipiscing elit Ut et massa mi. Aliquam in hendrerit urna. Pellentesque sit amet sapien fringilla, mattis ligula
```

Comment:
There's some lorem ipsum here, don't we want this replaced or dynamic?

Reply:
This will get replaced later when content gets filled